config.py

```diff
@@ -89,31 +89,46 @@
 
 DIVERT_TO_PRIVATE = ('help', )
 
-ROOMS_TO_JOIN = (
+ROOMS_TO_JOIN = [
     'coala',
-    'coala/offtopic',
-    'cobot-test',
-    'corobo',
-    'devops',
-    'community',
-    'coala/gsoc',
-    'coala/maintainers',
     'coala-bears',
-    'bearship',
-    'gci',
-    'cobot',
-    'performance',
-    'documentation',
-    'conferences',
-    'coala/workshops',
-    'coala/artwork-corner',
-    'freelancers',
-    'editor-plugins',
+    'corobo',
     'depman',
-    'ast',
-    'aspects',
-    'community'
-)
+]
+
+if BACKEND == 'Gitter':
+    ROOMS_TO_JOIN += [
+        'aspects',
+        'ast',
+        'bearship',
+        'coala',
+        'coala/artwork-corner',
+        'coala/gsoc',
+        'coala/maintainers',
+        'coala/offtopic',
+        'coala/workshops',
+        'cobot',
+        'cobot-test',
+        'community',
+        'community',
+        'conferences',
+        'devops',
+        'documentation',
+        'editor-plugins',
+        'freelancers',
+        'gci',
+        'performance'
+    )
```

Comment:
Can you move this https://github.com/coala/corobo/blob/master/config.py#L119 here?

Reply:
This `)` is a syntax error.  A list ends with a `]`

Reply:
Also, don't forget to add a trailing comma.